doc/ci.md

```diff
@@ -13,6 +13,7 @@ Please ask if you need login data.
 ## Concurrent Builds
 
 Concurrent builds on the same branch are disabled.
+Only the newest commit will run.
 The master branch is excluded from this rule.
```

Comment:
@tschawax Is the text below maybe more clear?

> FCFS? LIFO?

Do you mean related to your own pushes (should now be clarified below) or how Jenkins internally works (in which order the jobs are executed also between the branches?)

> That means we have to manually trigger a skipped build?

No, manual trigger is only needed on CI problems, which are luckily rare.

```suggestion
## Succeeding Pushes

If you push to branches:

- Only the newest commit will be built.
  It does not matter if commits in between actually build.
- Builds of older pushes are aborted.
- Also `jenkins build please` will abort previous builds and start the build again.
- The latest PR build job will stay deployed on (https://pr.permaplant.net).
- The [master branch](https://github.com/ElektraInitiative/PermaplanT/commits/master) is excluded from these rules.
```